tests/profiler/test_profiler.py

```diff
@@ -495,6 +495,7 @@ def look_for_trace(trace_dir):
     # Wrap the logger in a list so it becomes a LoggerCollection
     logger = [TensorBoardLogger(save_dir=tmpdir)]
     trainer = Trainer(default_root_dir=tmpdir, profiler="pytorch", logger=logger, limit_train_batches=5, max_epochs=1)
+    # TODO: What behavior do we want once trainer.logger is deprecated?
```

Comment:
How should this unit test be handled? Part of this issue https://github.com/PyTorchLightning/pytorch-lightning/pull/11731/files#r801162619